cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -18,11 +18,31 @@
     test_requirements.quota,
     pytest.mark.meta(server_roles="+automate"),
     test_requirements.vm_migrate,
-    pytest.mark.usefixtures('setup_provider', 'uses_infra_providers'),
+    pytest.mark.usefixtures('setup_provider'),
     pytest.mark.provider([VMwareProvider, RHEVMProvider], scope="module", selector=ONE_PER_TYPE)
 ]
 
 
+@pytest.fixture
+def set_default(provider, request):
+    """This fixture is used to return paths for provisioning_entry_point, reconfigure_entry_point
```

Comment:
Thanks for the detailed description, but it doesn't make sense to me that this is a fixture. Its not doing any setup, its simply returning a string, or a formatted string.

This could just be a statement within the `catalog_item` fixture, with `set_default` being changed to a more descriptive indirect parameter, like `default_entry_point`. 

Is there value to testing both of these settings for any test using a catalog item? Consider actually parametrizing this fixture directly.

Reply:
@mshriver 
- I am agree with the statement that ```Its not doing any setup; its simply returning a string, or a formatted string.```
- But this needs to be different fixture because as I mentioned in the doc string of ```set_default``` fixture; In future some test cases may require all the three custom entry points. **Also I have called ```provider``` fixture from this fixture.**
- Parametrising this fixture directly may not be good idea if we want to pass all the three entry points to ```catalog item``` fixture at a time.
- Right now, there are  only two test cases for this scenario(custom entry points) [one for cloud](https://github.com/ManageIQ/integration_tests/pull/7903/files) and [other one for infra providers](https://github.com/ManageIQ/integration_tests/pull/8112/files)
- I would like to discuss about ```set_default``` fixture more in future for such a scenarios.